src/plugins/timeline/plugin.js

```diff
@@ -40,15 +41,14 @@ export default function () {
       cssClass: 'icon-timeline',
       initialize: function (domainObject) {
         domainObject.composition = [];
-        domainObject.configuration = {
-          useIndependentTime: false
-        };
+        domainObject.configuration = configuration;
```

Comment:
So, I _think_ this is going to cause all new timeline objects to reference the same configuration object because of Webpack caching. This could cause configuration to bleed between instances. Configuration.js needs to export a factory function that is invoked here, so we get a new configuration object each time.

Reply:
Good catch. Corrected this issue.